libraries/AP_Scripting/docs/docs.lua

```diff
@@ -1367,6 +1367,21 @@ function mission:state() end
 ---@return boolean
 function mission:cmd_has_location(cmd)end
 
+-- desc
+---@param tag integer
+---@return boolean
+function mission:jump_to_tag(tag) end
+
+-- desc
+---@param tag integer
+---@return integer
+function mission:get_index_of_jump_tag(tag) end
+
+-- desc
+---@return integer|nil
+---@return integer|nil
+function mission:get_last_jump_tag() end
```

Comment:
This documentation is wrong.

Reply:
@IamPete1 please help

Reply:
If it passes CI you should be good. If it doesn't it should tell you what it expects.

Reply:
it passes CI so resolving